lib/facter/ssh_client_version.rb

```diff
@@ -1,19 +1,23 @@
 Facter.add("ssh_client_version_full") do
   confine :kernel => [ 'Linux' , 'SunOS' , 'FreeBSD' , 'Darwin' ]
-  setcode do
-    version = Facter::Util::Resolution.exec('sshd -V 2>&1').
-      lines.
-      to_a.
-      select { |line| line.match(/^OpenSSH_/) }.
-      first.
-      rstrip
 
-    version.gsub(/^OpenSSH_([^ ]+).*$/, '\1')
+  version = Facter::Util::Resolution.exec('sshd -V 2>&1').
```

Comment:
Could we also replace `sshd` with `ssh` here? If we're talking about the client, might as well use the client version.